docs/samples/browser-plugin-meetings/app.js

```diff
@@ -409,23 +414,23 @@ function verifyPassword() {
     meeting
       .verifyPassword(joinOptions.pin, joinOptions.captcha)
       .then((res) => {
-        if (res.failureReason === 'NONE') {
+        /* we have a bug in the SDK code: for case if wbxappapi requires only captcha and no password */
+        if (res.isPasswordValid) {
           passwordCaptchaStatusElm.innerText = 'Password is verified';
           passwordCaptchaStatusElm.style.backgroundColor = '#49e849';
           verifyPasswordElm.disabled = true;
           document.getElementById('btn-join').disabled = false;
           document.getElementById('btn-join-media').disabled = false;
-          // joinMeetingNow();
-        } else if (res.failureReason === "WRONG_PASSWORD") {
-          passwordCaptchaStatusElm.innerText = 'Password is required';
-          passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
-        } else if (res.failureReason === 'WRONG_CAPTCHA') {
+        } else if (!res.isPasswordValid && res.requiredCaptcha) {
           passwordCaptchaStatusElm.innerText = 'Password & Captcha is required';
           passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
           meetingsJoinCaptchaImgElm.src = res.requiredCaptcha.verificationImageURL;
           meetingsJoinCaptchaImgElm.hidden = false;
           meetingsJoinCaptchaElm.type = "text";
           refreshCaptchaElm.hidden = false;
+        } else if (!res.isPasswordValid) {
+          passwordCaptchaStatusElm.innerText = 'Password is required';
+          passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
         } else {
           passwordCaptchaStatusElm.innerText = 'Check the logs for more info';
```

Comment:
This else block looks like unreachable code. For eg, if `isPasswordValid` is true then it will return from first if condition and if `false or undefined` then will return from 3rd condition.

Reply:
`if (res.isPasswordValid) {
  passwordCaptchaStatusElm.innerText = 'Password is verified';
  passwordCaptchaStatusElm.style.backgroundColor = '#49e849';
  verifyPasswordElm.disabled = true;
  document.getElementById('btn-join').disabled = false;
  document.getElementById('btn-join-media').disabled = false;
} else if (res.requiredCaptcha) {
  passwordCaptchaStatusElm.innerText = 'Password & Captcha is required';
  passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
  meetingsJoinCaptchaImgElm.src = res.requiredCaptcha.verificationImageURL;
  meetingsJoinCaptchaImgElm.hidden = false;
  meetingsJoinCaptchaElm.type = "text";
  refreshCaptchaElm.hidden = false;
} else {
  passwordCaptchaStatusElm.innerText = 'Password is required';
  passwordCaptchaStatusElm.style.backgroundColor = '#fa6e6e';
}`

This could be the final block. WDYT?

Reply:
Looks good. @meta-dipanshu-sharma